examples/keyvalue/gen-go/keyvalue/tchan-keyvalue.go

```diff
@@ -94,6 +96,41 @@ func (s *tchanAdminServer) Methods() []string {
 	}
 }
 
+// RegisterInterceptors registers the provided interceptors with the server.
+func (s *tchanAdminServer) RegisterInterceptors(interceptors ...thrift.Interceptor) {
+	if s.interceptors == nil {
+		interceptorsLength := len(interceptors)
+		s.interceptors = make([]thrift.Interceptor, interceptorsLength, interceptorsLength)
+	}
+
+	s.interceptors = append(s.interceptors, interceptors...)
```

Comment:
Yeah this was fixed in the actual template (this is generated code here), but I don't think i re ran thrift-gen, which is why this has the old version.